plugins/prometheus/README.md

```diff
@@ -36,7 +36,7 @@ PLUGINS = ['prometheus']
 The below settings are configured with reasonable defaults:
 
 ```python
-ALERTMANAGER_API_URL = 'http://localhost:9093'
+ALERTMANAGER_API_URL = [ 'http://localhost:9093' ]
```

Comment:
This needs to be backwards compatible with existing config (ie. accept a single value or a list).